net/openr/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openr
+PKG_SOURCE_DATE:=2019-06-14
+PKG_SOURCE_VERSION:=79dab3d3c1bd612052d7b61e1aa01c289ed177d9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_DATE).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/facebook/openr/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_HASH:=133a15d0cd7fe48bce9a922c9fe2854bfe3104c6cbde94db595a3c6d108b923f
+
+PKG_MAINTAINER:=Amol Bhave <ambhave@fb.com>
+
+PKG_BUILD_DEPENDS:=fbthrift/host
+PKG_BUILD_DIR:=$(BUILD_DIR)/openr-$(PKG_SOURCE_VERSION)
+PKG_BUILD_PARALLEL:=1
+CMAKE_BINARY_SUBDIR:=build
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/openr
+	SECTION:=network
+	CATEGORY:=Network
+	TITLE:=OpenR: Distributed platform for building autonomic network functions.
+	DEPENDS:=+libatomic +libzmq-nc +libnl-core +libnl-route +re2 +fbzmq
+endef
+
+define Package/openr/description
+	Open Routing, OpenR, is Facebook's internally designed and developed routing
+	protocol/platform. Originally built for performing routing on the Terragraph
+	network, its has found adoption in other networks at Facebook including
+	Express Backbone.
+endef
+
+TARGET_LDFLAGS:=-lboost_program_options -lboost_regex -lboost_context -lstdc++ -latomic
```

Comment:
This doesn't look right. Are you sure it's needed?

Reply:
It doesn't link without these options. Throws undefined symbols error. Spent some time scratching my head, I'll try digging more. This solution seemed to work.

Reply:
-lstdc++ specifically should be included by default. Not sure about the others.